numpy/linalg/linalg.py

```diff
@@ -150,6 +151,8 @@ def _commonType(*arrays):
             rt = double
         if rt is double:
             result_type = double
+        elif rt is float16:
+            result_type = float16
```

Comment:
This isn't correct - this will make `_commonType(arr_f64, arr_f16)` return `(double, f16)`

Reply:
@eric-wieser sorry for the overlook, will modify it immediately